foundry.toml

```diff
@@ -3,14 +3,19 @@ src = "src"
 out = "out"
 libs = ["lib"]
 optimizer = true
-optimizer_runs = 100
+optimizer_runs = 10000
 bytecode_hash = "none"
 odyssey = true
 ffi = true
 
 remappings = [
     "webauthn-sol/=lib/webauthn-sol/"
 ]
+additional_compiler_profiles = [{ name = "via_ir", optimizer_runs = 10000, via_ir = true }]
+
+compilation_restrictions = [
+  { paths = "src/MinimalDelegationEntry.sol", via_ir = true }
+]
```

Comment:
so this makes it so only the entry.sol is compiled w ir, not the tests?

Reply:
yep!